Gems/Multiplayer/Code/Source/Editor/MultiplayerEditorSystemComponent.cpp

```diff
@@ -401,25 +332,20 @@ namespace Multiplayer
         AZ::Interface<IMultiplayerEditorConnectionViewportMessage>::Get()->DisplayMessage(("(3/3) " + sending_leveldata_message).c_str());
         AZ_Printf("MultiplayerEditor", sending_leveldata_message.c_str())
 
-        const auto& allAssetData = prefabEditorEntityOwnershipInterface->GetPlayInEditorAssetData();
 
         AZStd::vector<uint8_t> buffer;
         AZ::IO::ByteContainerStream byteStream(&buffer);
 
         // Serialize Asset information and AssetData into a potentially large buffer
-        for (auto& [spawnableName, spawnableAssetData] : allAssetData)
+        for (const auto& preAliasedSpawnableData : m_preAliasedSpawnablesForServer)
         {
-            for (auto& asset : spawnableAssetData.m_assets)
-            {
-                AZ::Data::AssetId assetId = asset.GetId();
-                AZStd::string assetHint = asset.GetHint();
-                auto hintSize = aznumeric_cast<uint32_t>(assetHint.size());
-
-                byteStream.Write(sizeof(AZ::Data::AssetId), reinterpret_cast<void*>(&assetId));
-                byteStream.Write(sizeof(uint32_t), reinterpret_cast<void*>(&hintSize));
-                byteStream.Write(assetHint.size(), assetHint.data());
-                AZ::Utils::SaveObjectToStream(byteStream, AZ::DataStream::ST_BINARY, asset.GetData(), asset.GetData()->GetType());
-            }
+            // This is an un-aliased level spawnable (example: Root.spawnable and Root.network.spawnable) which we'll send to the server
+            auto hintSize = aznumeric_cast<uint32_t>(preAliasedSpawnableData.assetHint.size());
+
+            byteStream.Write(sizeof(AZ::Data::AssetId), reinterpret_cast<const void*>(&preAliasedSpawnableData.assetId));
+            byteStream.Write(sizeof(uint32_t), reinterpret_cast<void*>(&hintSize));
```

Comment:
Nitpick: the `reinterpret_cast` to `void*` shouldn't be needed.

Reply:
Fixed!